integ-test/src/test/java/org/opensearch/sql/ppl/DateTimeFunctionIT.java

```diff
@@ -254,141 +236,136 @@ public void testDateAdd() throws IOException {
 
   @Test
   public void testDateTime() throws IOException {
-    JSONObject result =
```

Comment:
Please fix idents, you should finally have no changes in that function (test)

Reply:
indents fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/336/commits/5ae58b2c177d29b861902bb8f001d28b416e1ec3, and updated to pass

Reply:
Not fixed sorry

Reply:
My apologies, it should be updated now in https://github.com/Bit-Quill/opensearch-project-sql/pull/336/commits/b37c31d941ed14e9b3c864454c2ba503ea779a65

Reply:
Not yet...

Reply:
Alright I believe that the spacing issues are now fixed, the issue was caused by some jankiness on my part but with the test spotless merged in we should not be having those problems anymore